mne/time_frequency/psd.py

```diff
@@ -120,13 +129,14 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, n_fft=256,
         Max frequency of interest
     n_fft : int
         The length of the tapers ie. the windows. The smaller
-        it is the smoother are the PSDs.
-    pad_to : int | None
-        The number of points to which the data segment is padded when
-        performing the FFT. If None, pad_to equals `n_fft`.
+        it is the smoother are the PSDs. The default value is 256.
+        If ``n_fft > len(epochs.times)``, it will be adjusted down to
+        ``len(epochs.times)``.
     n_overlap : int
-        The number of points of overlap between blocks. The default value
-        is 0 (no overlap).
+        The number of points of overlap between blocks. Will be adjusted
+        to be <= n_fft.
+    proj : bool
+        Apply SSP projection vectors.
```

Comment:
FWIW I ran a quick test adapted from our docstring checking test, and out of 6048 public functions, we appear to have 17640 arguments that have default values. The word "default" (case-insensitive) shows up 3318 times (< 20% at best), so those docstrings appear to be in a minority currently.